### PR TITLE
V3.begin combined scripts dev

### PR DESCRIPTION
**Pull request by alexdelin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,28 +5,45 @@
...
-    <div id="theBid"></div>
+      <p>Bid Count: </p>
+      <div id="bidCount"></div>
+      
+    </div>
+    <p>Bidder: </p>
+    <div id="theBidder"></div>
 
     <p>
       <a href="" onclick="shuffle();return false;">Shuffle Deck</a>
```
Comment (alexdelin) [inline index.html]: if you change this from <a ..... into <button ... [to make styling easier] it fails calling the functions shuffle() and deal(). Why?

Comment (sshilal1) [inline index.html]: dont change my code u fuck

Comment (sshilal1) [inline index.html]: also dont use those, they were there for reference but I dont use them anymore

Comment (sshilal1) [inline index.html]: Actually, Ill just remove them now, new commit inc

Comment (msosa) [inline index.html]: lol it should work
http://www.w3schools.com/tags/tryit.asp?filename=tryhtml_button_test

try and add an alert like its doing there... and remove that return false;

File: cards.js [hunk 2]
```diff
@@ -405,19 +405,30 @@ function stackCardCount() {
...
+    return power.toString();
+  }
+  else if (power == 11) {return "J";}
+  else if (power == 12) {return "Q";}
+  else if (power == 13) {return "K";}
+  else if (power == 14) {return "A";}
+}
+
+function convertRtoP(rank) {
+  if(!isNaN(rank) {
```
Comment (alexdelin) [inline cards.js]: gotta close both those parentheses bro

**Pull request merged**